README.md

```
@@ -645,29 +646,91 @@ When visualization is needed, add the package reference as follows in your proje
-> This is a limitation of Qudi's scanning approach, but it should be sufficient for most cases.
+### Registration Status Visualization
+When visualization is enabled, visual runtime errors will be output when there are issues in the registration, such as missing registrations or circular dependencies. This helps you identify and resolve problems in your registration.
 
-#### (TODO) Report Missing Registrations
-When registrations are missing for interfaces in your project, a visual runtime error like the following is output:
+#### Missing Registrations
+When registrations are missing for interfaces in your project, a visual error like the following is output:
 
-```
-TODO
+![](./assets/missing.png)
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)